src/db/tools/services/reports/pages_users_to_main_service.py

```
@@ -0,0 +1,83 @@
+def get_pages_users_to_main(record_id: int) -> PagesUsersToMainRecord | None:
+    """Get a pages_users_to_main record by ID."""
+    with get_session() as session:
+        orm_obj = session.get(PagesUsersToMainRecord, record_id)
+        if not orm_obj:
+            logger.warning(f"PagesUsersToMain record with ID {record_id} not found")
+            return None
+        return orm_obj
+
+
+def add_pages_users_to_main(
+    id: int | None = None,
```

Ruff (0.15.20)

[error] Function argument `id` is shadowing a Python builtin

(A002)